.github/workflows/publish-gem.yml

```diff
@@ -0,0 +1,17 @@
+name: publish-gem
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - name: Release Gem
+        uses: dawidd6/action-publish-gem@v1
```

Comment:
If you wanna save yourself a dependency you can copy from here:

https://github.com/chanzuckerberg/redcord/blob/master/.github/workflows/publish.yml

and of course

https://github.com/chanzuckerberg/redcord/blob/master/bin/publish-gem.sh

Reply:
Oh, I like this! I'll implement it.